app/views/home.njk

```diff
@@ -1,31 +1,35 @@
 {% extends '_layout.njk' %}
 
-{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
-
-{% block beforeContent %}
-  {{ govukBreadcrumbs({
-    items: [
-      {
-        text: "Home",
-        href: "#"
-      }]
-  }) }}
-{% endblock %}
-
 {% block content %}
   <div class="govuk-grid-row">
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-          <h1 class="govuk-heading-l">{{ serviceName }}</h1>
-          <p class="govuk-body">Manage payment services</p>
-          <p class="govuk-body"><a href="/payment-holds" class="govuk-link">View payment holds</a></p>
-          <p class="govuk-body"><a href="/add-payment-hold" class="govuk-link">Add payment hold</a></p>
-          <p class="govuk-body"><a href="/remove-payment-hold" class="govuk-link">Remove payment hold</a></p>
-           <p class="govuk-body"><a href="/payment-schemes" class="govuk-link">View payment scheme</a></p>
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-xl">{{ serviceName }}</h1>
+
+      <div class="govuk-cards govuk-cards--2">
+        <div class="govuk-cards--2__item govuk-card--dashboard">
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Payment holds</h2>
+          <p class="govuk-heading-xl govuk-!-margin-bottom-5">
+            {{ totalHolds }}
+          </p>
+          <ul class="govuk-list">
+            <li>
+              <a href="/payment-holds" class="govuk-link">Manage holds</a>
+            </li>
+          </ul>
+        </div>
+        <div class="govuk-cards--2__item govuk-card--dashboard">
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Schemes</h2>
+          <p class="govuk-heading-xl govuk-!-margin-bottom-5">
+            {{ totalSchemes }}
```

Comment:
Total schemes is showing zero when there are 2 schemes.

Reply:
Fixed as part of above